package.json

```
@@ -4,6 +4,11 @@
   "description": "This is a fairly functional work-in-progress. Let's try to keep it clean and functionally minimal as we evolve it.",
   "main": "content.js",
   "scripts": {
+    "clean": "rimraf dist",
+    "predev": "npm run clean",
+    "dev": "cross-env NODE_ENV=development parcel watch src/content.js src/hotReload.js ",
+    "prebuild": "npm run clean",
+    "build": "cross-env NODE_ENV=production parcel build src/content.js",
```

As we're using `yarn` as the package manager I'd say we also use yarn to run the scripts here as not to cause confusion.
```suggestion
    "predev": "yarn run clean",
    "dev": "cross-env NODE_ENV=development parcel watch src/content.js src/hotReload.js ",
    "prebuild": "yarn run clean",
    "build": "cross-env NODE_ENV=production parcel build src/content.js",
```

---

Thanks for the suggestion @abenettt!

Although we use `yarn`, `npm` is always available as it ships with node.js. Folks installing the unpacked extension (the extension is currently not published) might not have `yarn` on their machine.

`yarn` is only required if someone is working on the extension and adds a new dependency.

---

The way I understand it you still have to install the dependencies to build the extension and as we're using `yarn` as a package manager, you will need to use it as `npm` does not know how to use the `yarn.lock` file. Also in the `README.md` we explicitly state that `yarn` has to be installed to use this extension.

---

@abenettt, it's true, that npm is not aware of yarn.lock, but it would still be able to install. Regardless, you are right about what's said in the readme, which is funny because I updated it to say that yarn is a requirement. 🙃

It's not a technical reason, but I've never seen a project use the yarn command in scripts, just using it to run npm scripts. Happy to make the change, I'm just wondering if it's standard to use `yarn` in the npm scripts. I through out a poll to see https://twitter.com/nickytonline/status/1300265591119532033

---

I thought about it poll or no poll and consistency makes sense. Thanks for the suggestion @abenettt !

---

@abenettt, since we're making things uniform here, if you're up for it, we can do the same in the package.json for Forem as well. https://github.com/forem/forem/blob/master/package.json#L21